pybamm/solvers/casadi_solver.py

```diff
@@ -4,6 +4,22 @@
 import casadi
 import pybamm
 import numpy as np
+from scipy.interpolate import interp1d
```

Comment:
looks like this is out of date with the changes to casadi, but should be fixed by merging

Reply:
Ah thanks, I was wondering why the tests were failing